### DISCUSSION TITLE
Why i got a red arrow when i define a date/time field inside the filter

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2853] [Q&A]
I define the LastModifiedTime as part of our filters, but we got a red arrow with the following message:-

![image](https://user-images.githubusercontent.com/79726511/225256634-9d8a94f3-b96a-427d-8dd0-3144d513a722.png)

so what this message is trying to communicate to us?

**Comment by wobba** [comment]
Hard to know without knowing your exact web part setup and connection. Remember that a filter web part and result web part must be connected both ways.

**Reply by mvcsharepointdev** [reply]
@wobha yes they are connected

**Reply by mvcsharepointdev** [reply]
@wobba  but seems we will get this message on any filter which has a Date Range as its type

**Reply by wobba** [reply]
I checked it, and it's ok. It's just a warning that the filter is static with manual to/from values. Nothing you need to think about.
